akka-actor-typed-tests/src/test/java/jdocs/akka/typed/OOIntroTest.java

```diff
@@ -181,16 +181,18 @@ private Gabbler() {}
           .onMessage(
               ChatRoom.MessagePosted.class,
               (context, message) -> {
-                System.out.println(
-                    "message has been posted by '" + message.screenName + "': " + message.message);
+                context
+                    .getLog()
+                    .info(
+                        "message has been posted by '{}': {}", message.screenName, message.message);
                 return Behaviors.stopped();
```

Comment:
a few in IntroSpec and OOIntroSpec too

Reply:
My grepping skills are useless :(

Reply:
Ah think i was just looking in Java at the time